app/src/main/java/org/apache/fineract/ui/online/customers/customerprofile/CustomerProfileActivity.java

```diff
@@ -164,6 +168,16 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
                     Toaster.show(findViewById(android.R.id.content),
                             getString(R.string.permission_denied_write));
                 }
+            case ConstantKeys.PERMISSIONS_REQUEST_CAMERA: {switch (requestCode) {
+                case ConstantKeys.PERMISSION_REQUEST_WRITE_EXTERNAL_STORAGE: {
```

Comment:
why we are using two switches? It is too complicated, even by that, as I see in both cases we are doing the same thing. Instead, add only  `case ConstantKeys.PERMISSION_REQUEST_WRITE_EXTERNAL_STORAGE:` before `case ConstantKeys.PERMISSIONS_REQUEST_CAMERA:` at line number 163 without adding `break` after the first case. It will do the same work as your boilerplate codes are doing at line numbers[171-180].